__tests__/integration/utils/wallet-helper.js

```diff
@@ -16,6 +16,17 @@ export class WalletHelper {
    */
   #words;
 
+  /**
+   * seedKey referencing a preconfigured multisig on the wallet-headless
+   */
+  #seedKey;
+
+  /**
+   * boolean indicating this is a MultiSig Wallet
+   * XXX; We currently do not support starting a multisig wallet from `words`
+   */
```

Comment:
```suggestion
   * XXX; We currently do not support starting a multisig wallet from `words`
   * @type {boolean}
   */
```

I suggest specifying the variable type to help static analyzers understand our code, while we don't use typescript.